docs/application/native/guides/connectivity/overview.md

```diff
@@ -18,10 +18,14 @@ You can use smart card functionalities to access secure elements. You can open a
 You can connect to a Wireless Local Area Network (WLAN) and transfer data over the network. You can activate and deactivate a local Wi-Fi device, and to connect to a WLAN network in the infrastructure mode.
 
 - [Wi-Fi Direct&reg;](wifi-direct-n.md) **in mobile applications only**
-You can find nearby Wi-Fi Direct devices and discover their details. You can also form a Wi-Fi Direct group to communicate over a peer-to-peer link without wireless access points (base stations) in the infrastructure mode.
+Find Wi-Fi Direct devices, discover their details, and form Wi-Fi Direct group to communicate over a peer-to-peer link without wireless access points (base stations) in the infrastructure mode.
+
+- [STC (Smart Traffic Control)](stc.md)
+Statistics about network data consumption is fetched by system and applications.
```

Comment:
**Rephrase to:**  The system and applications monitor the network data consumption statistics.

Reply:
STC module always monitors network traffic and user has no control over its start and stop.

There are two users for this module:-
1. normal application developers - User can only fetch/retrieve network usage statistics, so that it can show graphs or use those statistics as per application's use case.

Network Statistics consists of foreground and background data usage for below items:-
1. System level data usage (inbound and outbound)
2. Interface level data usage (inbound and outbound)
3. Application level data usage (inbound and outbound)